tests/integration/conftest.py

```diff
@@ -22,7 +22,7 @@ def mock_fabric_run(monkeypatch):
     )
 
 
-def _get_free_port(upper_bound=50_000):
+def _get_free_port(upper_bound=90_000):
```

Comment:
I had managed to revert it back before all the other commits, but I failed to remember at the last one...
Let me revert it again. I prefer to have issues on my laptop rather than the CI.